biodatacatalyst-ui/src/main/webapp/picsureui/overrides/dataset/dataset-view.js

```diff
@@ -1,60 +1,19 @@
-define([], function() {
+define(["underscore", "dataset/utilities"], function(_, dataUtils) {
```

Comment:
What is underscore for?

Reply:
I think that addition is outdated-- I needed it in one of the overrides, but decided to just add conditionals to the core function instead of overriding it here. I'll remove.